e2e/native-esm/__tests__/native-esm.test.js

```diff
@@ -10,17 +10,19 @@ import dns from 'dns';
 // eslint-disable-next-line no-restricted-imports
 import {readFileSync} from 'fs';
 import {createRequire} from 'module';
+import prefixDns from 'node:dns';
```

Comment:
I assume this is due to #12007 - can you just roll back the change CI is unhappy about?